modules/mono/csharp_script.h

```diff
@@ -161,6 +161,9 @@ class CSharpScript : public Script {
 		Vector<DocData::ClassDoc> docs;
 		return docs;
 	}
+	virtual String get_class_icon_path() const override {
+		return String();
```

Comment:
Now that https://github.com/godotengine/godot/pull/72619 has been merged.

```suggestion
		return icon_path;
```